src/Behat/Context.php

```diff
@@ -629,35 +583,33 @@ private function assertSelectedValue(NodeElement $element, string $value, string
      */
     private function assertInputValue(NodeElement $element, string $value, string $selector = 'input'): void
     {
-        $input = $element->find('css', $selector);
-        if (!$input) {
-            throw new Exception('Input not found in selector: ' . $selector);
-        }
-        $inputValue = $input->getValue();
-        if ($inputValue !== $value) {
+        if ($this->assertElementInElement($element, $selector)->getValue() !== $value) {
```

Comment:
`assertElementInElement` is getter, it should be named so then and made `protected` so it can be used when context is extended

Reply:
Make sense. Done!